mozilla-services/consvc-shepherd
# [SHEP-116] filtering searchability on the campaigns table

## References

[SHEP-116](https://mozilla-hub.atlassian.net/browse/SHEP-116)

## Problem Statement

Filters need to be added to the campaign page. This is one of the refinements based on the demo feedback.

## Proposed Changes

1. Added filters to the campaign page in the React UI.
2. Updated the Advertiser API to use the newly created table.
3. Updated docstrings to ensure consistency.
4. Included the name in the search filter on the campaign overview page since the campaign names were merged into the main.

## Verification Steps

1. Run the Ads Ops React Dashboard locally.
2. Navigate to the campaigns page.
3. Check the filters and search functionality for campaigns.
4. Check the search functionality on the campaigns overview page to ensure it works for the Name column.

## Visuals

https://github.com/user-attachments/assets/ae642a28-8129-4e12-ba4c-4edbdce206f1
